index.js

```diff
@@ -178,6 +179,16 @@ function getMinimumSecurityLevel(serviceOrOptions?: string | Options) {
   return specifiedLevel || SECURITY_LEVEL.ANY;
 }
 
+function getUseStrongBoxOption(serviceOrOptions?: string | Options) {
+  var useStrongBox = true;
+
+  if (typeof serviceOrOptions === 'object') {
+    useStrongBox = serviceOrOptions.useStrongBox;
```

Comment:
if service options is `{ securityLevel: SECURE_HARDWARE }` then this will return undefined, but you want to return a boolean I think.